### PR TITLE
Cargo Voidfill

### PR DESCRIPTION
**Pull request by BlueHNT**
## About The Pull Request
Added a few crates and missing parts of kits that matter.

## Why It's Good For The Game
Adding the MP armor to buyable with the missing 1911.

## Changelog
:cl:
add: Added MP armor crate
add: Added 1911 crates (Single, Triple, 6Magazine + 2Ammobox)
tweak: Added ballistic helmets to ballistic armor shipment

/:cl:

### CODE REVIEW

File: code/modules/cargo/packs.dm
Comment (someone543) [inline code/modules/cargo/packs.dm:1]: We already put most of our crate packs in [nsv13/code/modules/cargo/packs.dm](https://github.com/BeeStation/NSV13/blob/d719872e230576f4234bb25217794f8fce061075/nsv13/code/modules/cargo/packs.dm). Probably best to move everything here to there

Comment (BlueHNT) [reply code/modules/cargo/packs.dm:1]: Sure, gimme a moment.

**Comment by SerynEngi** [comment]
I'd comment it on the individual files itself, but I have a comment on basically everything, so I'll just make it a list.

1. There are 3 hardsuits in the game designed to be properly resistant to fire, and only one that is player facing that is immune to acid or the players being damaged by the acid.

**There is absolutely no good reason to raise the fire or acid resistance of any of these armors.**

2. You more than doubled resistance, in some cases, to bullet damage, but did not lower the laser resistance down. If laser's are so rare (and they are) it would make sense that the armors would not be built to be much, if at all, resistant to it.

3. Raising bomb resistance is just gaming against ship-to-ship combat. Antags of all sort on NSV use explosives very infrequently, and always have, due to their relatively unreliable nature. and there is once again no actual reason to change these from their normal values.

4. The combat shoes in general looks like a blatant attempt to stack resistances, particularly against environmental. I disagree with this item's existence as a whole.

5. Modified jackboots. Same as point 4, no reason for it to exist other than "I want to be stronger". I disagree with it as a whole.

6. I do not understand why you lowered the acid resistance of the reflective armor. Thematic isn't a real reason to change gameplay design. Gameplay is a reason to change gameplay design.

**In conclusion: most of this PR seems geared to make security the strongest geared in the game. While I personally agree with _swapping_ laser and bullet resistance, I think you need to remember that security hardsuits are geared to deal with personnel threats and NOT environmental hazards.**

**Comment by BlueHNT** [comment]
> 

2 of the things you listed are literally unchanged. Bro, read after yourself. While I do see some notions, I just flat-out disagree.

Mainly, you can't buy up new hard suits, so they were meant to be undestroyable as an item.

Resistances to bullets were increased due to the game being literally PvE and you only ever fight against boarders and syndie boarders. Thus they were changed to roughly match them while still being somewhat inferior.
_There is a PvP game mode that plays out once per 2 months, but its always ship vs ship anyway.._

Bomb resistance most of the time matches up to bullet by roughly half to 3/4 and I still kept it down mostly.

Combat shoes are a knockoff of the real combat shoes, made to poke fun as most of the stuff we got is a knockoff of syndicate gear in some way or another. That's just how NT plays.

Modified jackboots are literally unchanged, I just gave it a funny name and description, lmao

Reflective armor, needed to pull it down somewhat as it's the only laser-proof armor and it had way too much everything. 


So all in all, I don't know who you are, but I really disagree with most of the shit you said

**Comment by BlueHNT** [comment]
Also, this was meant to balance the fact that the starting armor is literally better than most of these and changing it would be a mistake.

**Review by Bokkiewokkie — CHANGES_REQUESTED** [review]
If you want this PR to be considered only keep the M1911 cargo crates and make them more expensive

**Comment by BlueHNT** [comment]
> If you want this PR to be considered only keep the M1911 cargo crates and make them more expensive

Having someone redo the armor values completely while actually referencing all the uses, and current issues and balancing it together with the PvE and PvP game modes might be a better idea than me trying to make them not shit on the spot.

File: code/modules/cargo/packs.dm [hunk 1]
```diff
@@ -550,11 +550,14 @@
...
-	cost = 1200
+	desc = "Contains three sets of bulletproof armors with their corresponding helmets. Guaranteed to reduce a bullet's stopping power by over half. Requires Armory access to open."
+	cost = 3000
 	contains = list(/obj/item/clothing/suit/armor/bulletproof,
 					/obj/item/clothing/suit/armor/bulletproof,
-					/obj/item/clothing/suit/armor/bulletproof)
+					/obj/item/clothing/suit/armor/bulletproof,
+					/obj/item/clothing/head/helmet/alt,
+					/obj/item/clothing/head/helmet/alt,
+					/obj/item/clothing/head/helmet/alt)
```
Comment (Bokkiewokkie) [inline code/modules/cargo/packs.dm:560]: Needs an NSV tag

File: code/modules/projectiles/boxes_magazines/ammo_boxes.dm [hunk 2]
```diff
@@ -75,6 +75,9 @@
 	ammo_type = /obj/item/ammo_casing/c45
 	max_ammo = 20
 
+obj/item/ammo_box/c45/m45 // 1911 version with 3 loads
+	max_ammo = 24
+
```
Comment (Bokkiewokkie) [inline code/modules/projectiles/boxes_magazines/ammo_boxes.dm:80]: This should be moved to a file in the nsv13 folder

File: nsv13/code/modules/cargo/packs.dm [hunk 3]
```diff
@@ -531,6 +531,52 @@
...
+					/obj/item/clothing/suit/ship/squad/military_police,
+					/obj/item/clothing/suit/ship/squad/military_police,
+					/obj/item/clothing/under/ship/military_police,
+					/obj/item/clothing/under/ship/military_police,
+					/obj/item/clothing/under/ship/military_police)
+	crate_name = "MP armor crate"
+
+/datum/supply_pack/security/armory/m45single
+	name = "M1911 handgun Single-pack"
+	desc = "For the ones which know that .45 ACP is a superior cartridge"
```
Comment (Bokkiewokkie) [inline nsv13/code/modules/cargo/packs.dm:548]: All of these crates should actually describe what is in them instead of being the same text copied three times

**Review by Bokkiewokkie — CHANGES_REQUESTED** [review]

File: code/modules/clothing/shoes/miscellaneous.dm [hunk 4]
```diff
@@ -134,6 +134,8 @@
 	pocket_storage_component_path = /datum/component/storage/concrete/pockets/shoes
 
 /obj/item/clothing/shoes/jackboots/fast
+	name = "modified jackboots"
+	desc = "Nanotrasen-issue Security combat boots for combat scenarios or combat situations. This pair seems to be modified with lighter materials."
```
[the remark this answers is not in the document]
Comment (BlueHNT) [reply code/modules/clothing/shoes/miscellaneous.dm:138]: Only person who gets these is the EXO from mail tho, if there were more classes like sec and such who had some way to access em, it would make more sense to make it less NT. I'ma do the fixes tho.

Comment (BlueHNT) [inline code/modules/clothing/shoes/miscellaneous.dm:138]:
```suggestion
/obj/item/clothing/shoes/jackboots/fast // NSV13 - EXO mail-boots can be told apart
	name = "modified jackboots"
	desc = "Security combat boots for combat scenarios or combat situations. This pair seems to be modified with lighter materials."
```

[the remark this answers is not in the document]
Comment (BlueHNT) [reply code/modules/clothing/shoes/miscellaneous.dm:138]:
```suggestion
	name = "modified jackboots" // NSV13 - EXO mail-boots can be told apart
	desc = "Security combat boots for combat scenarios or combat situations. This pair seems to be modified with lighter materials."
```

File: nsv13/code/modules/cargo/packs.dm (hunk 3 above)
Comment (BlueHNT) [reply nsv13/code/modules/cargo/packs.dm]: Fixed with [98bb89c](https://github.com/BeeStation/NSV13/pull/2458/commits/98bb89c993757938cb0bea21b6e6ed411155d707)

File: code/modules/projectiles/boxes_magazines/ammo_boxes.dm (hunk 2 above)
Comment (BlueHNT) [reply code/modules/projectiles/boxes_magazines/ammo_boxes.dm]: fixed with [98bb89c](https://github.com/BeeStation/NSV13/pull/2458/commits/98bb89c993757938cb0bea21b6e6ed411155d707)

File: code/modules/cargo/packs.dm (hunk 1 above)
Comment (BlueHNT) [reply code/modules/cargo/packs.dm:561]: added the tag in [98bb89c](https://github.com/BeeStation/NSV13/pull/2458/commits/98bb89c993757938cb0bea21b6e6ed411155d707)

File: nsv13/code/game/objects/items/custom_guns.dm [hunk 5]
```diff
@@ -38,16 +38,16 @@
...
 	icon = 'nsv13/icons/obj/projectiles_nsv.dmi'
 	icon_state = "pdc"
 
 /obj/item/projectile/bullet/peacekeeper/stun
 	name = "6mm electro-shock round"
-	damage = 6 //Straight up WAY worse than the security glock at stunning, but they serve a purpose
+	damage = 4 //Straight up WAY worse than the security glock at stunning, but they serve a purpose it used to kill a bit too often
 	armour_penetration = 0 //Less strong. Can't penetrate armor.
-	stamina = 15 //Slightly more than 3 bursts to stun
+	stamina = 20 //Slightly more than 3 bursts to stun Minor buff to make it viable
```
[the remark this answers is not in the document]
Comment (BlueHNT) [reply nsv13/code/game/objects/items/custom_guns.dm]: Quite sad since that gun is horrible, but understandable. Values returned to original in [98bb89c].(https://github.com/BeeStation/NSV13/pull/2458/commits/98bb89c993757938cb0bea21b6e6ed411155d707)

File: code/modules/clothing/shoes/miscellaneous.dm (hunk 4 above)
[the remark this answers is not in the document]
Comment (BlueHNT) [reply code/modules/clothing/shoes/miscellaneous.dm]: suggested in [98bb89c](https://github.com/BeeStation/NSV13/pull/2458/commits/98bb89c993757938cb0bea21b6e6ed411155d707)

File: nsv13/code/game/objects/items/custom_guns.dm [hunk 6]
```diff
@@ -47,7 +47,7 @@
 	name = "6mm electro-shock round"
 	damage = 6 //Straight up WAY worse than the security glock at stunning, but they serve a purpose
 	armour_penetration = 0 //Less strong. Can't penetrate armor.
-	stamina = 15 //Slightly more than 3 bursts to stun
+	stamina = 15 //Slightly more than 3 bursts to stun Minor buff to make it viable
```
Comment (BlueHNT) [inline nsv13/code/game/objects/items/custom_guns.dm:50]: comment nuke
```suggestion
	stamina = 15 //Slightly more than 3 bursts to stun
```

File: code/modules/clothing/shoes/miscellaneous.dm [hunk 7]
```diff
@@ -133,7 +133,10 @@
 	permeability_coefficient = 0.05 //Thick soles, and covers the ankle
 	pocket_storage_component_path = /datum/component/storage/concrete/pockets/shoes
 
+// NSV13 modified
 /obj/item/clothing/shoes/jackboots/fast
+	name = "modified jackboots"
+	desc = "Security combat boots for combat scenarios or combat situations. This pair seems to be modified with lighter materials."
```
Comment (Bokkiewokkie) [inline code/modules/clothing/shoes/miscellaneous.dm:139]: You can put this change in an NSV file as well. Otherwise it can probably also just be ported upstream.

File: nsv13/code/modules/cargo/packs.dm [hunk 8]
```diff
@@ -531,6 +531,52 @@
 					/obj/item/gun/energy/laser)
 	crate_name = "laser crate"
 
+/datum/supply_pack/security/mparmor
+	name = "Military Police Replacement Crate"
+	desc = "Three vests of MP armor vests complete with MP undershirts, everything to replace which fire razed to the ground."
+	cost = 3600
+	contains = list(/obj/item/clothing/suit/ship/squad/military_police,
+					/obj/item/clothing/suit/ship/squad/military_police,
+					/obj/item/clothing/suit/ship/squad/military_police,
```
Comment (Bokkiewokkie) [inline nsv13/code/modules/cargo/packs.dm:540]: no helmets?

**Review by Bokkiewokkie — CHANGES_REQUESTED** [review]
Fun fact all of the

File: code/modules/clothing/shoes/miscellaneous.dm (hunk 7 above)
Comment (BlueHNT) [reply code/modules/clothing/shoes/miscellaneous.dm:139]: port

File: nsv13/code/modules/cargo/packs.dm (hunk 8 above)
Comment (BlueHNT) [reply nsv13/code/modules/cargo/packs.dm:540]: There aren't MP helmets to accompany the MP armors as far as I know. I could add in the berets however.

**Review by Bokkiewokkie — APPROVED** [review]
